examples/all-clusters-app/linux/BUILD.gn

```diff
@@ -64,6 +64,7 @@ source_set("chip-all-clusters-common") {
     "${chip_root}/examples/all-clusters-app/all-clusters-common/src/tcc-mode.cpp",
     "${chip_root}/examples/all-clusters-app/all-clusters-common/src/water-heater-mode.cpp",
     "${chip_root}/examples/all-clusters-app/linux/diagnostic-logs-provider-delegate-impl.cpp",
+    "${chip_root}/examples/all-clusters-app/linux/thermostat-delegate-impl.cpp",
```

Comment:
Why not just use the impl from thermostat-app, just like we're using things from examples/energy-management-app, lock-app (see the chip-lock-app-common source-set below), etc?

Reply:
done